src/ansys/hps/client/jms/api/jms_api.py

```diff
@@ -81,14 +82,26 @@ def fs_url(self) -> str:
             self._fs_url = _find_available_fs_url(self.get_storage())
         return self._fs_url
 
-    def get_api_info(self):
+    def get_api_info(self) -> dict:
         """Get information of the JMS API that the client is connected to.
```

Comment:
```suggestion
        """Get information on the JMS API that the client is connected to.
```
"of the JMS API" didn't read well. Hope "on the JMS API" is OK.